### ISSUE TITLE
Lose Formatting (Global Fonts Elementor) / Possible Cause - Background Fallback

### ISSUE THREAD

**Issue by jmedia123** [elementor/elementor#12410]
## Prerequisites

- [X ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by whokneadsdough** [comment]
See #12363

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.12

Feel free to update

Thanks!
